### DISCUSSION TITLE
큰숲백과 goToTop 구현 변경

### DISCUSSION THREAD

**Discussion by ghost** [hsl0/mediawiki-skin-float#4] [General]
큰숲백과 goToTop.js 구현이 [여기](https://bigforest.a2hosted.com/wiki/%EB%AF%B8%EB%94%94%EC%96%B4%EC%9C%84%ED%82%A4:Gadget-NavigationButtons.js)로 이전되었습니다. #buttons의 id를 지우고 class에 navigation-buttons을 넣었습니다. 일부 기존 코드가 불필요해져서 연락 드려야 할 것 같았습니다.

**Comment by hsl0** [comment]
goToTop.css는 float 스킨 전용으로 계속 쓰나 보네요. 다른 스킨에서 잘 작동되는지 한번 확인해 보세요. Vector-2022에 float 스킨을 활성화한 경우가 아니면 버튼이 공중에 그냥 떠있습니다.

**Reply by ghost** [reply]
gototop 관련 css에서 #buttons만 사용하고 #buttons > button은 이제 사용하고 있지 않습니다.

**Reply by ghost** [reply]
네 다른 스킨에서 박스 없이 버튼만 달랑 뜨네요. 조만간 수정해야겠습니다

**Reply by ghost** [reply]
> gototop 관련 css에서 #buttons만 사용하고 #buttons > button은 이제 사용하고 있지 않습니다.

#buttons를 .navigation-buttons로 교체해주시면 좋겠습니다.

**Reply by hsl0** [reply]
GitHub 사용법도 익힐 겸 직접 PR해보시는 건 어떨까요?

**Reply by ghost** [reply]
네 네 가능합니다. 제가 혼자서 수정하고 커밋하고 푸시하는거 밖에 해보지 않았네요..
